packages/synapse-react-client/src/components/SynapseChat/SynapseChat.tsx

```diff
@@ -122,8 +136,8 @@ export const SynapseChat: React.FunctionComponent<SynapseChatProps> = ({
       setInteractions([...interactions, pendingInteraction])
       setCurrentResponse('')
       setCurrentResponseError('')
-      setCurrentProgressMessage('')
```

Comment:
Move progress UI into SynapseChat (from SynapseChatInteraction), and look at the trace events instead of the async job status